src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -106,8 +123,11 @@ case class SubmissionValidationHeader(
 // result of an expression parse
 case class SubmissionValidationValue(
```

Comment:
we are now recording these in the database so it has to be a domain object with an id field